priv/protos/persistent.proto

```diff
@@ -89,14 +93,60 @@ message CreateReq {
 	Options options = 1;
 
 	message Options {
-		event_store.client.shared.StreamIdentifier stream_identifier = 1;
+		oneof stream_option {
+			StreamOptions stream = 4;
+			AllOptions all = 5;
+		}
+		event_store.client.StreamIdentifier stream_identifier = 1 [deprecated=true];
```

Comment:
the `[dreprecated=true]` symbol has me worried that eventually we'll need to either support multiple major versions of spear or do some smart on-the-fly protobuf-module selection magic :grimacing: